docs/user/openstack/observability.md

```diff
@@ -0,0 +1,349 @@
+
+> [!NOTE]
+Make sure you have the Cluster Observability Operator installed in the OpenShift cluster running the OpenStack control plane, as this is a requirement for the OpenStack Telemetry Operator. Follow [these directions](https://github.com/openstack-k8s-operators/architecture/blob/main/examples/dt/uni01alpha/control-plane.md#cluster-observability-operator) to install it.
+
+To check that the telemetry machinery is correctly installed, you can issue this command:
+
+```bash
+oc -n openstack get monitoringstacks metric-storage -o yaml
+```
+
+The `monitoringstacks` CRD being installed is a good indicator that telemetry is functional.
+
+
+<!--
+```bash
+oc patch OpenStackControlPlane/controlplane --type merge -p '{"spec":{"telemetry":{"enabled": true, "template":{"ceilometer":{"enabled": true}}}}}'
+```
+-->
+
+Before configuring remote-write in RHOSO's telemetry operator, create a secret
+in the `openstack namespace` containing the HTTPS client certificates for
+authenticating to Prometheus. We'll call it `mtls-bundle`:
+
+```bash
+oc --namespace openstack \
+    create secret generic mtls-bundle \
+        --from-file=./ca.crt \
+        --from-file=osp-client.crt \
+        --from-file=osp-client.key
+```
+
+Then, edit the controlplane configuration to setup the metric storage:
+
+```bash
+oc edit openstackcontrolplane/controlplane
+```
+
+We will configure RHOSO's telemetry operator to write metrics to our external
+Prometheus instance.
+
+Look for the `metricStorage` stanza. It can be found at the
+`.spec.telemetry.template.metricStorage` path. We will need to use
+a `customMonitoringStack` structure that cannot coexist with the
+`monitoringStack` one. Replace the `metricStorage` structure with one that
+looks like this:
+
+```yaml
+      metricStorage:
+        customMonitoringStack:
+          alertmanagerConfig:
+            disabled: false
+          logLevel: info
+          prometheusConfig:
+            scrapeInterval: 30s
+            remoteWrite:
+            - url: https://external-prometheus.example/api/v1/write
+              tlsConfig:
+                ca:
+                  secret:
+                    name: mtls-bundle
+                    key: ca.crt
+                cert:
+                  secret:
+                    name: mtls-bundle
+                    key: ocp-client.crt
+                keySecret:
+                  name: mtls-bundle
+                  key: ocp-client.key
+            replicas: 2
+          resourceSelector:
+            matchLabels:
+              service: metricStorage
+          resources:
+            limits:
+              cpu: 500m
+              memory: 512Mi
+            requests:
+              cpu: 100m
+              memory: 256Mi
+          retention: 1h
+        dashboardsEnabled: false
+        dataplaneNetwork: ctlplane
+        enabled: true
+        prometheusTls: {}
+```
+
+After saving the file and letting the change propagate, verify that you receive
```

Comment:
might be worth mentioning that the external Prometheus is https://external-prometheus.example

Reply:
it's there at around line 70